dot-env.template

```diff
@@ -52,4 +52,7 @@ OPENPIPE_API_KEY=...
 # Tavus
 TAVUS_API_KEY=...
 TAVUS_REPLICA_ID=...
-TAVUS_PERSONA_ID=...
\ No newline at end of file
+TAVUS_PERSONA_ID=...
+
+#Krisp
+KRISP_MODEL_PATH=...
```

Comment:
Easy way to choose which model we wish to use.

Reply:
I don't see any new environment variables...

Reply:
We are using [it here](https://github.com/pipecat-ai/pipecat/blob/af867502f440d091ad75b7728ca86a5a9a1d2058/src/pipecat/audio/filters/krisp_filter.py#L40), in case a model path is not provided.